cumulusci/cli/flow.py

```diff
@@ -170,3 +168,5 @@ def flow_run(runtime, flow_name, org, delete_org, debug, o, skip, no_prompt):
                     "Scratch org deletion failed.  Ignoring the error below to complete the flow:"
                 )
                 click.echo(str(e))
+
+    runtime.alert(f"Flow Complete: {flow_name}")
```

Comment:
@Br4nd0R sorry for the back and forth on this but I just realized we also should call `runtime.alert` with a different message when there is an error. It's the thing that causes an OS notification to pop up and we wouldn't want to lose that when there's an error.

Reply:
no problem